aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -309,6 +317,14 @@ public String s3FileIoSseKey() {
     return s3FileIoSseKey;
   }
 
+  public int s3DeletionBatchSize() {
+    return s3DeletionBatchSize;
+  }
+
+  public void setS3DeletionBatchSize(int deletionBatchSize) {
+    this.s3DeletionBatchSize = deletionBatchSize;
```

Comment:
need to add validation here

Reply:
curious to know, shall we allow it also to be set from aws properties  with string k/v Map?

Reply:
Yes it definitley should. In my latest revision I have added a property!